src/MangaStreamParser.ts

```diff
@@ -317,9 +316,17 @@ export class MangaStreamParser {
         }
 
         image = image?.split('?resize')[0] ?? ''
-        image = image.replace(/^\/\//, 'https://')
-        image = image.replace(/^\//, 'https:/')
 
+        if (!image?.startsWith('http')) {
```

Comment:
Why is this changed? Pretty sure it worked fine before hand, if not give some context on which item it didn't work for.